bedrock/firefox/templates/firefox/personal.html

```diff
@@ -0,0 +1,55 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% extends "base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Firefox Web browser')}} - {% endblock %}
```

Comment:
Just a note that the campaign landing page will be en-us only.

On Sunday, May 10, 2015, Francesco Lodolo <notifications@github.com> wrote:

> In bedrock/firefox/templates/firefox/personal.html
> <https://github.com/mozilla/bedrock/pull/2969#discussion_r30004986>:
>
> > @@ -0,0 +1,55 @@
> > +{# This Source Code Form is subject to the terms of the Mozilla Public
> > + # License, v. 2.0. If a copy of the MPL was not distributed with this
> > + # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
> > +
> > + {% extends "base-resp.html" %}
> > +
> > +{% block page_title_prefix %}{{_('Firefox Web browser')}} - {% endblock %}
>
> I would mostly ask to confirm that's wanted. The other string is not in a
> shared file, so we'd still need it in the new lang file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/bedrock/pull/2969/files#r30004986>.
>


-- 
Cory Price
/ckprice